.github/workflows/bi-connectors.yml

```diff
@@ -11,40 +11,41 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Pack Tableau JDBC connector
       id: pack-tableau-jdbc
       run: |
         zip -r opensearch_sql_jdbc.taco . -x *.taco
       working-directory: bi-connectors/TableauConnector/opensearch_sql_jdbc
     - name: Prepare Power BI ODBC connector
       run: |
-        cp OpenSearch.pq OpenSearch.m
+        cp OpenSearchProject-ODBC.pq OpenSearchProject-ODBC.m
       working-directory: bi-connectors/PowerBIConnector
     - name: Pack Power BI ODBC connector
       id: pack-powerbi-odbc-os-proj
       run: |
-        zip OpenSearchProject.mez *.png *.m *.resx *.pqm
+        zip OpenSearchProject-ODBC.mez *.png *.m *.resx *.pqm
       working-directory: bi-connectors/PowerBIConnector
     - name: Prepare Power BI ODBC connector for 'Amazon OpenSearch Service'
       id: prep-powerbi-odbc-amz-os-svc
       run: |
+        mv OpenSearchProject-ODBC.m AmazonOpenSearchService-ODBC.m
         sed -i 's/<value>OpenSearch Project<\/value>/<value>Amazon OpenSearch Service<\/value>/g' resources.resx
-        sed -i 's/Documentation.Name = "OpenSearch Project"/Documentation.Name = "Amazon OpenSearch Service"/g' OpenSearch.m
-        sed -i 's/section OpenSearch;/section AmazonOpenSearchConnector;/' OpenSearch.m
-        sed -i 's/\[DataSource.Kind="OpenSearch", Publish="OpenSearch.Publish"\]/[DataSource.Kind="AmazonOpenSearchConnector", Publish="AmazonOpenSearchConnector.Publish"]/g' OpenSearch.m
-        sed -i 's/OpenSearch.Contents/AmazonOpenSearchConnector.Contents/g' OpenSearch.m
-        sed -i 's/OpenSearch.Publish/AmazonOpenSearchConnector.Publish/g' OpenSearch.m
-        sed -i 's/OpenSearch.Icons/AmazonOpenSearchConnector.Icons/g' OpenSearch.m
-        sed -i 's/OpenSearchImpl/AmazonOpenSearchConnectorImpl/g' OpenSearch.m
-        sed -i 's/OpenSearchType/AmazonOpenSearchConnectorType/g' OpenSearch.m
-        sed -i 's/OpenSearch = \[/AmazonOpenSearchConnector = [/' OpenSearch.m
-#        sed -i 's/"opensearch-sql"/"Amazon-OpenSearch-Service"/g' OpenSearch.m
+        sed -i 's/Documentation.Name = "OpenSearch Project"/Documentation.Name = "Amazon OpenSearch Service"/g' AmazonOpenSearchService-ODBC.m
+        sed -i 's/section #"OpenSearchProject-ODBC";/section #"AmazonOpenSearchService-ODBC";/' AmazonOpenSearchService-ODBC.m
```

Comment:
Remove -ODBC except for the protocol within the connector.

Reply:
So OpenSearchProject and AmazonOpenSearchService. The main point was alignment to the Kind name and the Connector name inside the mez.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008